### DISCUSSION TITLE
Night Light not working in Regolith Desktop Ubuntu 22.10

### DISCUSSION THREAD

**Discussion by elioduarte** [regolith-linux/regolith-desktop#797] [Q&A]
Hi. First of all, big thanks to @kgilmer and all the other contributors. Regolith is my favorite desktop and I recommend it to all friends. 

I recently did a fresh install of Regolith 2.0/Ubuntu 22.10 on a new laptop using the [official iso](https://github.com/regolith-linux/regolith-ubuntu-iso-builder/releases/download/ubuntu-kinetic-2.2.0-20221211_050200/regolith-ubuntu-kinetic-2.2.0.zip), and I noticed the Gnome Night Light is not working. I also confirmed the Night Light is not working in the Gnome Classic Session, but is working in the Ubuntu Session. 

I'm not 100% sure if this is related, but It seems the `org.gnome.SettingsDaemon.Color.target` service is not initializing properly.

```bash
systemctl --user status org.gnome.SettingsDaemon.Color.target
○ org.gnome.SettingsDaemon.Color.target - GNOME color management target
     Loaded: loaded (/usr/lib/systemd/user/org.gnome.SettingsDaemon.Color.target; static)
     Active: inactive (dead)

Dez 24 11:45:07 hf systemd[2073]: Dependency failed for GNOME color management target.
Dez 24 11:45:07 hf systemd[2073]: org.gnome.SettingsDaemon.Color.target: Job org.gnome.SettingsDaemon.Color.target/start failed with result 'dependency'.
```

How can I investigate further to find the cause of this issue? Thanks.

**Comment by ap300** [comment]
I am experiencing this same issue.

Because regolith starts gnome-session with `--builtin` then I think the above systemd unit is never activated, and gnome-session launches gsd-color instead (it is running  in my session, and the old `pkill gsd-color` causes my session to crash)

I was able to get gsd-color running in verbose mode by modifying `/etc/xdg/autostart/org.gnome.SettingsDaemon.Color.desktop`, but in the verbose logging it appears as though gsd-color thinks it's operating correctly, yet no color is displayed.

I will continue to dig for a root cause but have yet to figure out where to look next.

**Comment by kgilmer** [comment]
I do not use this feature personally and haven't spent any time troubleshooting it.  I would first check with the `gnome-flashback` project if there are any specific work-arounds to the issue, as Regolith gets it's gnome integration from there.

**Comment by JohannesPertl** [comment]
Any update or workaround?

**Reply by peterrus** [reply]
Using https://gitlab.com/chinstrap/gammastep from Ubuntu 24.10 repos with good results. A workaround of course.
